addons/mrp_repair/tests/test_flows.py

```diff
@@ -0,0 +1,50 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import Form, tagged
+from odoo.addons.mrp.tests.common import TestMrpCommon
+from odoo.exceptions import AccessError
+
+
+@tagged('post_install', '-at_install')
+class TestMrpRepairFlows(TestMrpCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env.ref('base.group_user').write({'implied_ids': [(4, cls.env.ref('stock.group_production_lot').id)]})
```

Comment:
You're not using any lot here, so I don't get the reason why it would be useful ?
```suggestion
```